documentation/book/con-custom-resources-status.adoc

```diff
@@ -6,16 +6,53 @@
 
 = {ProductName} custom resource status
 
-The `status` property of a {ProductName}-specific custom resource publishes the _current_ state of the resource to users and tools that need the information.
+The `status` property of a {ProductName} custom resource publishes useful information about the resource to users and tools that need it.
 
-Status information is useful for tracking progress related to a resource achieving its _desired_ state, as defined by the `spec` property. The `status` provides the time and reason the state of the resource changed and details of events preventing or delaying the Operator from realizing the desired state.
+Several resources have a `status` property, as described in the following table.
+
+[cols="2*",options="header",stripes="none",separator=¦]
+|===
+
+¦{ProductName} Resource
+¦Publishes information on...
+
+m¦xref:type-KafkaStatus-reference[]
+¦The Kafka cluster.
+
+m¦xref:type-KafkaTopicStatus-reference[]
```

Comment:
Does this link already exist? If not we will need to wait until it is created by @tombentley before merging this for the CI pipelines.

Reply:
It might be better to remove/comment this and merge this PR now, than wait for a PR with `KafkaTopic.status`.

Reply:
I'll remove it and add a comment to the module.